make/jtreg.gmk

```diff
@@ -81,6 +81,7 @@ TARGETS.com.sun.javatest.regtest += $(BUILDDIR)/classes.java.lang.ok
 # resources required for com.sun.javatest.regtest
 
 RESOURCES.com.sun.javatest.regtest = \
+	$(CLASSDIR)/META-INF/services/java.util.spi.ToolProvider \
 	$(CLASSDIR)/com/sun/javatest/regtest/i18n.properties \
```

Comment:
This line is causing your conflict. It was recently removed, in 163ae `7903105: remove jtreg Ant task`